src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -766,17 +756,31 @@ public Geometry Location()
         {
             if (this.InternalElement.Location.GetType() == typeof(Autodesk.Revit.DB.LocationPoint))
             {
-                Autodesk.Revit.DB.LocationPoint pt = (Autodesk.Revit.DB.LocationPoint)this.InternalElement.Location;
-                return pt.Point.ToPoint(true);
+                Autodesk.Revit.DB.LocationPoint pt = this.InternalElement.Location as Autodesk.Revit.DB.LocationPoint;
```

Comment:
These lines are becoming too long, you can user `var pt = ...` , you can also define a new name for this type `using DBLocationPoint = Autodesk.Revit.DB.LocationPoint;`